src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -141,6 +143,26 @@ const formConfig = {
           ...formSignatureSchema,
           scrollAndFocusTarget,
         },
+        ohiScreen: {
+          path: 'champva-screen',
+          ...champvaScreenSchema,
```

Comment:
It might make sense to block this page on the review screen - it looks like if you change the value on review it creates a validation error without an obvious solution. 

```suggestion
          path: 'champva-screen',
          depends: () => !window?.location?.href?.includes('review-and-submit'),
          ...champvaScreenSchema,
```